src/main/java/com/github/princesslana/eriscasper/rest/Route.java

```diff
@@ -157,13 +159,36 @@ public int hashCode() {
     return get(path, noContent(), jsonResponse(rsClass));
   }
 
+  public static <Rq, Rs> Route<Rq, Rs> get(
+      String path, Class<Rq> rqClass, Function<Response, Rs> rsFunction) {
```

Comment:
I'm not excited at all in introducing every possible combo of Function/Class parameters that can be here. I'd rather have two: Class/Class, Function/Function. If the definition has to use a Function for one, having to call jsonRequestBody() there I think is ok.

Reply:
We should make those kinds of methods public then, because they're very useful.

Reply:
Yeah, I'm fine with that. I had been making them public as they were needed.

Reply:
gotcha, time to Remodel then